pkg.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This needs to be removed: https://github.com/mandiant/VM-Packages/pull/69/files#r968504190